packages/select/resources.js

```diff
@@ -1,20 +1,24 @@
 import {
   avProvidersApi,
+  avOrganizationsApi,
   avPermissionsApi,
   avNavigationApi,
   avUserApi,
   avCodesApi,
 } from '@availity/api-axios';
 import { ResourceSelect } from '.';
 import AvRegionSelect from './custom-resources/AvRegionSelect';
-import AvOrganizationSelect from './custom-resources/AvOrganizationSelect';
 
 const AvProviderSelect = ResourceSelect.create({
   resource: avProvidersApi,
   labelKey: 'uiDisplayName',
   requiredParams: ['customerId'],
   watchParams: ['customerId'],
 });
+const AvOrganizationSelect = ResourceSelect.create({
+  resource: avOrganizationsApi,
+  labelKey: 'name',
```

Comment:
I would add an arg to the `create` method with attributes that you want passed to the `postGetAttributes` prop on the `ResourceSelect`. This way the user can just passing `resourceIds` to `OrganizationSelect` and then just put that to the `additionalPostGetArgs` prop on the resource select for them as well as spreading whatever they may additionally pass at render time

Reply:
I think that would make it easier for the user by letting them pass in a prop/props vs wrapping those props in an object, but the `ResourceSelect.create` method will already spread the defaults and any passed in props on creation. I think in this case it's better to be explicit to the user by saying "if you put values in `additionalPostGetArgs` they will be supplied to the `postGet` method". This way would also allow any future changes to resources in `sdk-js` without us needing to modify a `postGetAttributes` for a resource in `availity-react`, we could just start passing new `additionalPostGetArgs` to the `ResourceSelect` immediately. Thoughts?